aion/web/elm/src/Update.elm

```diff
@@ -89,12 +91,15 @@ update msg model =
                 push_ =
                     Phoenix.Push.init "new:answer" ("rooms:" ++ (toString model.roomId))
                         |> Phoenix.Push.withPayload payload
+                        |> Phoenix.Push.onError (\v -> WrongAnswer)
```

Comment:
I have to agree with the point about scalability etc. In general the flow of the answerFeedback is totally the same, regardless of whether it is correct, almost correct or incorrect, therefore I would stick to sending the valuable information in one string and then displaying it differently depending on its content.

I believe we can do it as @pmrukot suggests or we could even send `evaluation` from elixir to elm and then assign certain strings on frontend side.